bokeh/plotting/helpers.py

```diff
@@ -552,15 +552,23 @@ def func(self, **kwargs):
         else:
             hglyph_ca = None
 
+        # handle the mute glyph, if any properties were given
+        if any(x.startswith('mute_') for x in kwargs):
+            mglyph_ca = _pop_colors_and_alpha(glyphclass, kwargs, prefix='mute_')
+        else:
+            mglyph_ca = None
+
```

Comment:
@mattpap just a heads up I am going to change this to `muted_` when I push docs changes